Sources/XCTest/XCTestCase.swift

```diff
@@ -39,8 +44,75 @@ public class XCTestCase: XCTest {
     ///   https://bugs.swift.org/browse/SR-1129 for details.
     public var _name: String
 
-    public required override init() {
-        _name = "\(self.dynamicType).<unknown>"
+    public override var testRunClass: AnyClass {
```

Comment:
`XCTestCase` is missing an override for `testCaseCount` (AFAICT it should just always `return 1`)